pt-BR/command_line.md

```diff
@@ -1,23 +1,25 @@
 **DO NOT READ THIS FILE ON GITHUB, GUIDES ARE PUBLISHED ON https://guides.rubyonrails.org.**
 
+Linhas de Comando do Rails
 The Rails Command Line
 ======================
 
-After reading this guide, you will know:
+Depois de ler este guia, você saberá:
 
-* How to create a Rails application.
-* How to generate models, controllers, database migrations, and unit tests.
-* How to start a development server.
+* Como criar uma aplicação Rails.
+* Como gerar modelos, controladores, migrações de banco de dados e testes de unidade.
+* Como começar a desenvolver um servidor.
+* Como experimentar objetos por meio de um shell interativo.
```

Comment:
```suggestion
* Como fazer experimentos com objetos por meio de um *shell* interativo.
```

Fiz uma leve sugestão mas ainda to muito na dúvida, o que você acha?

Fiquei na dúvida dessa tradução, experimentar me dá um sentido meio diferente de provar, aí fiz uma leve alteração.